krun.py

```diff
@@ -101,109 +103,90 @@ def sanity_check_user_change(platform):
 def create_arg_parser():
     """Create a parser to process command-line options.
     """
-    parser = argparse.ArgumentParser(description='Benchmark, running many fresh processes.')
```

Comment:
I don't currently have any rule in place. I actually had to search to see what the convention is.

There's no difference between `'` and `"` in python, but it seems people use `'` to express the notion of a string which is constant and `"` for format strings, which are *conceptually* expanded.

E.g.:

 * `'hello'`
 * `"hello %s" % name`

We could adopt such a notion if you like(?).